cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/OperatorsTest.scala

```diff
@@ -244,9 +252,94 @@ class OperatorsTest extends DocumentingTest {
           resultTable()
         }
       }
-      p("""Further information and examples regarding the use of regular expressions in filtering can be found in <<query-where-regex>>.
+      p(
+        """Further information and examples regarding the use of regular expressions in filtering can be found in <<query-where-regex>>.
           |In addition, refer to <<query-operator-comparison-string-specific>> for details on string-specific comparison operators.""")
     }
+    section("Temporal operators", "query-operators-temporal") {
+      p(
+        """Temporal operators comprise:
+          |
+          |* adding a <<cypher-temporal-durations, _Duration_>> to either a <<cypher-temporal-instants, temporal instant>> or another _Duration_: `+`
+          |* subtracting a _Duration_ from either a temporal instant or another _Duration_: `-`
+          |* multiplying a _Duration_ with a number: `*`
+          |* dividing a _Duration_ by a number: `/`
+          |
+          |The following table shows -- for each combination of operation and operand type -- the type of the value returned from the application of each temporal operator:
+          |
+          |[options="header"]
+          ||===
+          || Operator | Left-hand operand | Right-hand operand | Type of result
+          || <<syntax-add-subtract-duration-to-temporal-instant, `+`>>      | _Duration_/Temporal instant  | Temporal instant/_Duration_ | The type of the temporal instant
+          || <<syntax-add-subtract-duration-to-temporal-instant, `-`>>      | Temporal instant  | _Duration_ | The type of the temporal instant
+          || <<syntax-add-subtract-duration-to-duration, `+`>>      | _Duration_        | _Duration_ | _Duration_
+          || <<syntax-add-subtract-duration-to-duration, `-`>>      | _Duration_        | _Duration_ | _Duration_
+          || <<syntax-multiply-divide-duration-number, `*`>>      | _Duration_        | <<property-types, Number>> | _Duration_
+          || <<syntax-multiply-divide-duration-number, `/`>>      | _Duration_        | <<property-types, Number>> | _Duration_
+          ||===
+          |
+        """)
+      section("Adding and subtracting a _Duration_ to or from a temporal instant", "syntax-add-subtract-duration-to-temporal-instant") {
+        query(
+          """WITH datetime({year:1984, month:10, day:11, hour:12, minute:31, second:14, nanosecond: 1, timezone: '+01:00'}) AS aDateTime
```

Comment:
These queries have been very useful for testing, since they enable testing so many things at once. For documentation, I would rather have one example only per query (no UNWIND, no WITH, just a simple RETURN), which will be less overwhelming for users.